src/pages/Main/JobInfo/JobInfo.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import JOB_INFOS from '../JOB_INFOS';
+
+function JobInfo() {
+  return (
+    <Parent>
+      {JOB_INFOS.map(JOB => (
```

Comment:
```suggestion
      {JOB_INFOS.map(job => (
```

인자로 들어오는 값은 상수가 아니니까 일반적인 네이밍 컨벤션으로 적용하시는게 좋아보이네요!